src/pages/atomicdex/api/v20-dev/non_fungible_tokens/index.mdx

```diff
@@ -0,0 +1,610 @@
+| block_number         | integer          | Height of block in which transfer occurred.                                                      |
+| block_timestamp      | string           | Block date/time string.                                                                          |
+| chain                | string           | Chain name. One of `AVALANCHE`, `BSC`, `ETH`, `FANTOM`, or `POLYGON`.                            |
+| collection_name      | string           | May be `null`. Name of collection which token belongs to.                                        |
+| contract_type        | string           | Contract type. `ERC721` or `ERC1155`.                                                            |
+| token_address        | string           | Address of token transferred.                                                                    |
+| token_id             | string           | Token ID.                                                                                        |
+| possible_spam        | boolean          | May be `null`. If `true`, the contract address is has [been identified](https://docs.moralis.io/web3-data-api/evm/nft-spam-detection) as associated with spam, phishing attempts, or suspicious activities. |
+| transaction_hash     | string           | Transaction ID of the transfer.                                                                  |
+| transaction_index    | string           | Transaction index of the transfer.                                                               |
+| log_index            | string           | Log index of the transfer.                                                                       |
+| value                | string           | Tranfer value.                                                                                   |
+| transaction_type     | string           | Transation type. Possible values are `Single`.                                                   |
+| from_address         | string           | Address of previous owner which sent the token(s).                                               |
+| to_address           | string           | Address of new owner which received the token(s).                                                |
+| status               | string           | Transfer status. Will be either `Send` or `Receive`. When the `from_address` and `to_address` are the same (i.e. sending to yourself), this value will be `Receive`. |
+| verified             | integer          |                                                                                                  |
+| operator             | string           | May be `null`.                                                                                   |
+
+<DevComment>
+`block timestamp` is a date time string, though I would expect a unix timestamp
```

Comment:
Moralis provides it as ISO formatted string "block_timestamp": "2023-05-28T16:51:16.000Z", so I decided to live it as it is.

Reply:
k, comment removed